### PR TITLE
Drops references to the now-deprecated AllowTrust operation

### PR DESCRIPTION
**Pull request by Shaptic**
Additionally, introduce an example on how to use `SetTrustLineFlags`.

### CODE REVIEW

File: content/docs/issuing-assets/control-asset-access.mdx [hunk 1]
```diff
@@ -44,13 +44,13 @@ If the issuer wants to approve transactions on a case-by-base basis while allowi
 
 To intitiate a new operation, the holding account requests that the issuer approve and sign a transaction. Once the issuer inspects the operation and decides to approve it, they sandwich it between a set of operations, first granting authorization, then reducing it.  
 
-Here's a payment from A to B sandwiched between [`allow_trust`](../start/list-of-operations.mdx#allow-trust) operations:
+Here's a payment from A to B sandwiched between [`set_trust_line_flags`](../start/list-of-operations.mdx#allow-trust) operations:
```
Comment (Shaptic) [inline content/docs/issuing-assets/control-asset-access.mdx:47]:
```suggestion
Here's a payment from A to B sandwiched between [`set_trust_line_flags`](../start/list-of-operations.mdx#set-trustline-flags) operations:
```

File: content/docs/glossary/multisig.mdx [hunk 2]
```diff
@@ -87,9 +87,9 @@ This scheme is very flexible. You can require many signers to authorize payments
 
 ### Example 1: Anchors
 
-> You run an anchor that would like to keep its issuing key offline. That way, it's less likely a bad actor can get ahold of the anchor's key and start issuing credit improperly. However, your anchor needs to authorize people holding credit by running the `Allow Trust` operation. Before you issue credit to an account, you need to verify that account is OK.
+> You run an anchor that would like to keep its issuing key offline. That way, it's less likely a bad actor can get ahold of the anchor's key and start issuing credit improperly. However, your anchor needs to authorize people holding credit by running the [`Set Trust Line Flags` operation](../start/list-of-operations.mdx#set-trust-line-flags). Before you issue credit to an account, you need to verify that account is OK.
```
Comment (Shaptic) [inline content/docs/glossary/multisig.mdx:90]:
```suggestion
> You run an anchor that would like to keep its issuing key offline. That way, it's less likely a bad actor can get ahold of the anchor's key and start issuing credit improperly. However, your anchor needs to authorize people holding credit by running the [`Set Trust Line Flags` operation](../start/list-of-operations.mdx#set-trustline-flags). Before you issue credit to an account, you need to verify that account is OK.
```

File: content/docs/glossary/multisig.mdx [hunk 3]
```diff
@@ -28,7 +28,7 @@ Low Security:
 
 - [Transaction processing](./transactions.mdx)
   - Charging a fee or updating the sequence number for the source account
-- [Allow Trust](../start/list-of-operations.mdx#allow-trust) operation
+- [Allow Trust](../start/list-of-operations.mdx#allow-trust) and [Set Trust Line Flags](../start/list-of-operations.mdx#set-trust-line-flags) operations
```
Comment (Shaptic) [inline content/docs/glossary/multisig.mdx:31]:
```suggestion
- [Allow Trust](../start/list-of-operations.mdx#allow-trust) and [Set Trust Line Flags](../start/list-of-operations.mdx#set-trustline-flags) operations
```

**Review by rice2000 — APPROVED** [review]
This all looks great to me.  Thanks!

**Pull request merged**